config/config.go

```diff
@@ -51,6 +51,9 @@ const (
 
 	SettingDbUsername = "mongo_username"
 	SettingDbPassword = "mongo_password"
+
+	SettingLoginRateLimit        = "login_rate_limit_ms"
+	SettingLoginRateLimitDefault = "1s"
```

Comment:
I am unsure that will work, I think viper does not support units, does it?

Reply:
Yes, it does. It's using `time.ParseDuration` for duration-type configurations.

Reply:
I forgot to clean up the suffix (`_ms`) from before I converted it to a duration config. It should be fixed now.